lib/sentry_conn.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+class APIClient:
+    def __init__(self, base_url):
+        self.api_token = ''
+        self.project_slug = ''
+        if not base_url.endswith('/'):
+            base_url += '/'
+        self.__url = (
+                '{0}api/0/'
+            ).format(base_url)
+
+    def get(self, uri):
+        headers = {
+            'Authorization': f'Bearer {self.api_token}',
+            'Content-Type': 'application/json'
+        }
+        url = self.__url + uri
+        print(url)
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()  # Raise an error for bad status codes
+        return response.json()
```

Comment:
We would need pagination here to have a complete response

Reply:
It would be good to have a complete response. Currently, the API is limited to 100 items. There could be over 100 issues during the last day, but the developers seem to be interested in the *one* top issues in the notification. Getting 100 issues is more than enough for a health monitoring report.

If there is a reason to grab *all* issues during the past day, please let me know.